external-crates/move/crates/move-analyzer/tests/ide_testsuite.rs

```diff
@@ -48,12 +56,12 @@ impl UseDefTest {
         use_file: &str,
         use_file_path: &Path,
     ) -> anyhow::Result<()> {
-        // let file_name_mapping = &symbols.file_name_mapping;
         let def_info = &symbols.def_info;
         let UseDefTest { use_ndx, use_line } = self;
         writeln!(output, "-- test {test_idx} -------------------")?;
         writeln!(output, "use line: {use_line}, use_ndx: {use_ndx}")?;
-        let Some(uses) = mod_symbols.get(*use_line) else {
+        let lsp_use_line = use_line - 1; // 0th-based
+        let Some(uses) = mod_symbols.get(lsp_use_line) else {
```

Comment:
What is the motivation for changing these to be 1-indexed? Just to make it easier to write the tests?

Reply:
> What is the motivation for changing these to be 1-indexed? Just to make it easier to write the tests?

Yes. I found it quite annoying for what we call "line number" in the code and in config files to actually be "line number - 1" (and doing this calculation each time I was adding a new test).